data-esnext.js

```diff
@@ -3382,6 +3382,82 @@ exports.tests = [
   res : {
   }
 },
+{
+  name: 'BigInt',
+  category: STAGE3,
+  significance: 'medium',
+  spec: 'https://github.com/tc39/proposal-bigint',
+  subtests: [
+    {
+      name: 'basic functionality',
+      exec: function () {/*
+        return (1n + 2n) === 3n;
+      */},
+      res: {
+      },
+    },
+    {
+      name: 'constructor',
+      exec: function () {/*
+        return BigInt("3") === 3n;
+      */},
+      res: {
+      },
+    },
+    {
+      name: 'BigInt.asUintN',
+      exec: function () {/*
+        return typeof BigInt.asUintN === 'function';
```

Comment:
it'd be nice if all these tested the functions' behavior, not just their presence

Reply:
If I could test them, I could write a test, but right now I would kinda guess what the result is and therefore I only added the typeof test.

Reply:
@danez You can use Chrome 67 (Dev or Canary) to test now :)

Reply:
@ljharb Is this blocking the merge?

IMHO it would be better to have the data included in the table than to have nothing. We can always improve the tests in a follow-up commit. WDYT?